packages/runtime/src/controls/control-data-type-key.ts

```diff
@@ -0,0 +1 @@
+export const ControlDataTypeKey = '@@makeswift/type'
```

Comment:
Would love some input here, what the name of this file should be or where it should live.

Reply:
I think it's fine here for now. We can find a better place for it later.